### PR TITLE
feat(safes): add "book" handler

### PR DESCRIPTION
**Pull request by qtotuan**

### CODE REVIEW

File: src/v0/safes.ts [hunk 1]
```diff
@@ -156,4 +169,20 @@ export class Safes {
       }
     })
   }
+
+  book(body: BookRequestBody): Promise<SafeResponse> {
```
Comment (erezsob) [inline src/v0/safes.ts]: Because it's a post request I would rename the handler `setBook` to imply what it does.

File: src/v0/safes.ts [hunk 2]
```diff
@@ -46,6 +46,19 @@ export interface Safe {
...
+export interface BookRequestBody {
+  to: string,
+  from: string,
+  issuer: string
+  items: Array<Object>,
+  comment?: string,
+  initiated_at?: string
+}
+
+export interface BookResponse {
```
Comment (erezsob) [inline src/v0/safes.ts]: You don't use this type interface anywhere.

File: src/v0/safes.ts (hunk 1 above)
Comment (qtotuan) [reply src/v0/safes.ts]: Hmm, I wanted to be consistent with the other namings, that are also just one verb like "get" or "create". So "book" is actually following that logic.

File: src/v0/safes.ts (hunk 2 above)
Comment (qtotuan) [reply src/v0/safes.ts]: You are right, it is leftover code. I overlooked this.

File: src/v0/safes.ts (hunk 1 above)
Comment (erezsob) [reply src/v0/safes.ts]: Ohh, "book" is a verb in this case. I read it as a noun.

File: src/v0/safes.ts (hunk 2 above)
Comment (erezsob) [reply src/v0/safes.ts]: So just remember to remove it in your next changes to this file.

Comment (qtotuan) [reply src/v0/safes.ts]: Fixed in PR #176

**Pull request merged**